nomad/structs/structs_test.go

```diff
@@ -2226,14 +2226,18 @@ func TestReschedulePolicy_Validate(t *testing.T) {
 			},
 		},
 		{
-			desc: "Valid Unlimited config",
+			desc: "Ambiguous Unlimited config",
```

Comment:
This wording can be clarified- is `ambiguous` descriptive enough for this use case? Also now what is the valid case?